### DISCUSSION TITLE
When i run bin/composer install my vendor folder is still empty (mac M1)

### DISCUSSION THREAD

**Discussion by redouan-hajjari** [markshust/docker-magento#1150] [Q&A] [answered]
Hi Guys,

Im trying everything. but no success
When i open an exist project from bitbucket. and put it into the src folder.
Everything works fine. but when i do bin/composer install i see a list that installed all package.
But when i look at folder src/vendor i dont see any package. i thing its somewhere else. but now i cant start up the magento. 

Greetings,
Redouan

**Comment by YevhenZvieriev** [comment, marked as answer]
Hi @Redouan-Hajjari,

It is because `bin/composer install` executes within a docker container.

Note that this directory is not automatically synced for performance reasons, so you’ll need to run `bin/copyfromcontainer vendor` in order to see these classes.

**Reply by redouan-hajjari** [reply]
Thank you! it works!
